tasks/src/text-generation/about.md

```diff
@@ -26,6 +26,14 @@ The most popular models for this task are GPT-based models (such as [GPT-2](http
 
 These models are trained to learn the mapping between a pair of texts (e.g. translation from one language to another). The most popular variants of these models are [T5](https://huggingface.co/docs/transformers/model_doc/t5), [T0](https://huggingface.co/bigscience/T0pp) and [BART](https://huggingface.co/docs/transformers/model_doc/bart). Text-to-Text models are trained with multi-tasking capabilities, they can accomplish a wide range of tasks, including summarization, translation, and text classification.
 
+## Language Model Variants
+
+When it comes to text generation, the underlying language model can come in several types:
+
+- **Base models:** this refers plain language models like [T5](https://huggingface.co/t5-base), [OPT](https://huggingface.co/facebook/opt-30b), and [BLOOM](https://huggingface.co/bigscience/bloom). These models are good for fine-tuning and few-shot prompting, with [GPT-J](https://huggingface.co/EleutherAI/gpt-j-6B) and [GPT-NeoX](https://huggingface.co/EleutherAI/gpt-neox-20b) being among the best choices for domains involving English and code.
```

Comment:
```suggestion
- **Base models:** this refers plain language models like [T5](https://huggingface.co/t5-base), [OPT](https://huggingface.co/facebook/opt-30b), and [BLOOM](https://huggingface.co/bigscience/bloom). These models are good for fine-tuning and few-shot prompting, with [GPT-J](https://huggingface.co/EleutherAI/gpt-j-6B) and [GPT-NeoX](https://huggingface.co/EleutherAI/gpt-neox-20b) being among the best choices for domains involving English and code.
```
Can we somehow write a small glossary at top explaining fine-tuning and few-shot prompting and give small pointers?